### PR TITLE
update grafana datasource documentation

### PR DESCRIPTION
**Pull request by kerneljack**
Updating the wording as well as the screenshot depicting adding a Prometheus datasource to Grafana.

Signed-off-by: Khusro Jaleel <kerneljack@gmail.com>

### CODE REVIEW

File: content/docs/visualization/grafana.md [hunk 1]
```diff
@@ -25,15 +25,23 @@ By default, Grafana will be listening on
 
 ### Creating a Prometheus data source
 
-To create a Prometheus data source:
+First, we need to get Prometheus running and listening on port 9090. The easiest way to do this is via Docker:
```
Comment (brian-brazil) [inline content/docs/visualization/grafana.md:28]: These are docs to explain Grafana configuration, not docs on running Prometheus or using Docker. You can presume that Prometheus is already running.

File: content/docs/visualization/grafana.md [hunk 2]
```diff
@@ -25,15 +25,23 @@ By default, Grafana will be listening on
...
+
+1. Click on the "cogwheel" in the sidebar to open the Configuration menu.
+2. Click on "Data Sources".
+3. Click on "Add data source".
 4. Select "Prometheus" as the type.
-5. Set the appropriate Prometheus server URL (for example, `http://localhost:9090/`)
-6. Adjust other data source settings as desired (for example, turning the proxy access off).
-7. Click "Add" to save the new data source.
+5. Set the appropriate Prometheus server URL (in this case it's `http://localhost:9090/`)
+6. We need to use `Browser` mode because so that Grafana can connect to Prometheus directly.
```
Comment (brian-brazil) [inline content/docs/visualization/grafana.md:43]: This doesn't seem correct, and proxy mode is generally what is used. I'd leave this as-is.

File: content/docs/visualization/grafana.md (hunk 1 above)
Comment (kerneljack) [reply content/docs/visualization/grafana.md]: ok I have removed that section now

File: content/docs/visualization/grafana.md (hunk 2 above)
Comment (kerneljack) [reply content/docs/visualization/grafana.md]: I've changed back  now and used the correct wording as per current Grafana UI, which refers to Access methods.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**